build.gradle.kts

```diff
@@ -108,23 +114,21 @@ testing {
         val test by getting(JvmTestSuite::class) {
             useJUnitJupiter()
             dependencies {
-                implementation("org.springframework.boot:spring-boot-starter-test") {
+                implementation(libs.org.springframework.boot.spring.boot.starter.test) {
                     exclude("org.mockito", "mockito-core")
                     because("Use MockK instead of Mockito since it is better suited for Kotlin")
                 }
-                implementation("org.springframework.boot:spring-boot-starter-webflux")
-                implementation("com.ninja-squad:springmockk:4.0.0")
-                implementation("org.springframework.security:spring-security-test")
-                implementation("org.awaitility:awaitility:4.2.0")
-                implementation("org.jsoup:jsoup:1.16.1")
```

Comment:
I assume that both are not needed for the unit tests?

Reply:
Exactly, currently used only in integration tests.